src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriverCounterVisitor.java

```diff
@@ -16,17 +16,13 @@ public class DriverCounterVisitor implements DriverVisitor {
 
     private int count = 0;
 
+    public int getCount() {
+        return count;
+    }
+
     @Override
     public void visitDriverContainer(DriverContainer driverContainer) {
         count += 1;
-        for (Job2dDriver child : driverContainer.getChildren()) {
-            try {
-                Method acceptor = child.getClass().getMethod("accept", DriverVisitor.class);
-                acceptor.invoke(child, this);
-            } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
-                throw new RuntimeException(e);
-            }
-        }
     }
```

Comment:
This is incorrect. The Visitor should count inner drivers. Certainly, this is not stateless approach, because you depend on already existing value of count field.

Reply:
done